src/governance.cpp

```diff
@@ -1354,6 +1354,9 @@ void CGovernanceManager::RemoveInvalidProposalVotes()
         if (p.second->keyIDVoting != oldDmn->pdmnState->keyIDVoting) {
             changedKeyMNs.emplace_back(oldDmn->collateralOutpoint);
         }
+        if (p.second->pubKeyOperator != oldDmn->pdmnState->pubKeyOperator) {
```

Comment:
We now might end up adding the same MN twice. Changing this to `else if` should help